### ISSUE TITLE
Provide actual Python Code of the algorithm

### ISSUE THREAD

**Issue by mwittchen** [jdhotopp/tp.py-Direct-RNA-Sequencing-Manuscript-#1]
Hello,

this repository belongs to the publication "Deciphering transcript architectural complexity in bacteria and archaea" (https://doi.org/10.1128/mbio.02359-24) . You mention in the publication's abstract that you "[...] developed an algorithm for transcript prediction called tp.py using it to predict transcripts [...]". Nevertheless the algorithm is not published, there are no URL or other hints where to find the actual python algorithm. This repository only contains the R and Perl code for creating the diagrams of the publication.

So please **publish the python algorithm** for transcript prediction as this algorithm is the center piece of your publication!

All the best!

**Comment by adkinsrs** [comment]
Hi @mwittchen,

The algorithm can be found at https://github.com/Dunning-Hotopp-Lab/Direct_RNA_Sequencing and I have informed the maintainer of this repository to add the link in the README file.

**Comment by jdhotopp** [comment]
Thanks Shaun!
________________________________

**Comment by mwittchen** [comment]
Hi @adkinsrs ,

thank you for the link to the algorithm. I didn't find it in the publication. Did I miss it?

**Comment by adkinsrs** [comment]
No, this was on us.  I don't think any of us had noticed the "manuscript" repository (this one) was missing the algorithm link.

@jdhotopp can you add the algorithm link to the README for this repo to avoid future similar questions?  I do not have permissions to make edits to this repo (unless I fork and pull request).

**Comment by mwittchen** [comment]
Thank you very much!
